.travis.yml

```diff
@@ -6,7 +6,9 @@ before_script:
   - "sh -e /etc/init.d/xvfb start"
   - "npm install -g bower karma grunt-cli"
   - "bower install"
-script: grunt test
+script:
+  - grunt jshint
+  - grunt test
```

Comment:
instead of adding it here, it can also be added here to recognise errors locally 

```
  grunt.registerTask('server', [
    'concurrent:server',
    'copy:i18nTmp',
    'connect:dev',
    'jshint',
    'watch'
  ]);

  grunt.registerTask('test', [
    'jshint',
    'karma:all_tests'
  ]);
```